src/libs/antares/study/parameters.cpp

```diff
@@ -1114,6 +1124,22 @@ void Parameters::fixGenRefreshForNTC()
     }
 }
 
+void Parameters::fixGenRefreshForHydroMaxPower()
+{
+    if ((timeSeriesHydroMaxPower & timeSeriesToGenerate) != 0)
+    {
+        timeSeriesToGenerate &= ~timeSeriesHydroMaxPower;
+        logs.error() << "Time-series generation is not available for hydro max power. It "
+                        "will be automatically disabled.";
+    }
+    if ((timeSeriesHydroMaxPower & timeSeriesToRefresh) != 0)
+    {
+        timeSeriesToRefresh &= ~timeSeriesHydroMaxPower;
+        logs.error() << "Time-series refresh is not available for hydro max power. It will "
+                        "be automatically disabled.";
```

Comment:
```suggestion
        logs.warning() << "Time-series refresh is not available for hydro max power. It will "
                        "be automatically disabled.";
```